client/python/client.py

```diff
@@ -0,0 +1,284 @@
+import base64
+import time
+from urllib import response
+import grpc
+import uuid
+import threading
+from enum import Enum, auto
+from typing import Optional
+
+from armada.client import event_pb2
+from armada.client import event_pb2_grpc
+from armada.client import queue_pb2
+from armada.client import queue_pb2_grpc
+from armada.client import usage_pb2
+from armada.client import usage_pb2_grpc
+from armada.client import submit_pb2
+from armada.client import submit_pb2_grpc
+from k8s.io.api.core.v1 import generated_pb2 as core_v1
```

Comment:
I probably would avoid using internal k8 protobufs api in this code especially if we are using it to create pods/containers.  For unit/e2e testing, you could probably just use simple pod definitions without the k8 api.

Reply:
This is what I get for only looking at your client code.  It looks like our proto objects depend on this so the k8 is needed.  We don't need to reference it in client as I think the code generation of pods is verbose.  

But we can't avoid using internal k8 protobuf.

Reply:
I don't understand what you're proposing here. I need those imports in order to construct the proto msg that gets sent to armada, do I not?

Reply:
Yep.  Ignore this.